sre-operational-fabric.md

```diff
@@ -0,0 +1,119 @@
+- Networking: e.g. TCP/IP, UDP, ICMP, etc., MAC addresses, IP packets, DNS, SDN, OSI layers, and load balancing.
+- Strong expertise in designing, analyzing and troubleshooting large-scale distributed systems.
+- In depth knowledge of operating systems.
+- Familiarity with algorithms, data structures and complexity analysis.
+- Deep experience designing complex SaaS applications for cloud reliability and scalability.
+- Expert-level experience with GCP, AWS or Openstack APIs.
+- Deep experience with cloud infrastructure automation and CI/CD pipeline design.
+- Expertise in operational monitoring and management tools (Nagios, Datadog, etc.).
+- Advanced written & verbal communication skills, both highly technical and non-technical.
+- Ability to work closely with non-technical stakeholders and executives.
+- Capable of providing strategic technical advice.
+- Systematic problem-solving approach, coupled with a strong sense of ownership and drive.
+- Additional skills may be required depending on role; for example Kubernetes, Docker, Terraform, CEPH and other modern tools/technologies.
+
+### Job Complexity
+
+Supports high complexity deployments and internal teams on an as-needed basis.
+Responsible for the roll-out and operations of large scale, complex systems
+automation. Collaborates with other teams on tools for systems automation.
+Works in conjunction with multiple teams to ensure up-time and reliability of
+customer deployments.
+
+### Supervision
+
+Operates with little to no supervision. Provides technical and thought
+leadership to all levels of Infrastructure Services.
+
+### Experience Education
+
+High school diploma or equivalent required.  Bachelor's degree in Computer
+Science or equivalent experience. Usually requires 8+ years of information
+systems design/architecture/development experience. May require additional
+certifications depending on specialization.
+
+### Physical Demands
+
+General office environment. Considerable stress may occur at times. Some
+lifting up to 20 pounds required. General office environment.  May require long
+periods sitting and viewing a computer monitor.  Moderate levels of stress may
+occur at times.  No special physical demands required.
+
+## Focus
+
+As a Rackspace Private Cloud (RPC) Operational Fabric SRE, you will help
+support our organizational vision: To become the preferred brand for consuming
+complex open source infrastructure, as-a-Service, delivering industry-leading
+simplicity, efficiency, and reliability.
+
+Our team supports this mission by improving
+[observability](https://en.wikipedia.org/wiki/Observability) (i.e. insight
+into) and [controllability](https://en.wikipedia.org/wiki/Controllability)
+(i.e. state mutation) of cloud deployments located in datacenters throughout
+the world.
+
+### About Us
+
+- **Team Size**: Currently 8 senior developers/engineers and a technical manager
```

Comment:
s/8/7 right, unless there's dedicated QE? maybe i missed the response to this

Reply:
See today's agenda :)